### DISCUSSION TITLE
Would it be recommended to use pixijs for a image editor?

### DISCUSSION THREAD

**Discussion by nooblyf** [pixijs/pixijs#9605] [Q&A] [answered]
My use case is to create an image editor, something like [Canva](https://www.canva.com/)

Here's what i couldn't find on how to do with pixijs
1. Saving/Serializing the canvas (pixi application?) that anyone created to a database on server
2. Loading that saved instance to the app and being able to make further edits

Any help would be great! 😊

**Comment by bigtimebuddy** [comment, marked as answer]
Pixi is a great option for an image editor! There are many projects built with Pixi from drawing apps, photo editors, to game creators. That said, there is nothing out-of-the-box that allows for serializing and deserializing of the scene graph. You'll need to develop these bits yourself. Adding that functionality would be out of scope for the project, so it's definitely something that would need to live in user-land.

**Reply by nooblyf** [reply]
Got it, thanks.
